src/main/res/values/strings.xml

```diff
@@ -639,4 +639,5 @@ limitations under the License.
     <string name="field_set_primary">Primary</string>
     <string name="field_set_secondary">Secondary</string>
     <string name="field_remove_from_layout">Remove</string>
+    <string name="no_file_manager_found">No file manager found</string>
```

Comment:
No compatible file manager installed?

Reply:
Yes, that wording is better, changed it.